### PR TITLE
[EFv2] - Fix quote requests for ETH

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

@josojo can u try this please?

### CODE REVIEW

File: src/cow-react/api/gnosisProtocol/api.ts [hunk 1]
```diff
@@ -381,8 +381,16 @@ function _mapNewToLegacyParams(params: FeeQuoteParams): QuoteQuery {
     validTo,
     partiallyFillable: false,
     priceQuality,
+    // TODO: check this setup as this is for all other orders
```
Comment (josojo) [inline src/cow-react/api/gnosisProtocol/api.ts:384]: onchainOrder and signingScheme are optional. You can just leave them "unset" for the normal usecase.

**Comment by josojo** [comment]
Yes, looks good. I was also able to place and settle an order with this PR.

File: src/cow-react/api/gnosisProtocol/legacy/types.ts [hunk 2]
```diff
@@ -32,9 +32,11 @@ export type LegacyFeeQuoteParams = FeeQuoteParams & {
   chainId: ChainId
   priceQuality?: string
   isBestQuote?: boolean
+  isEthFlow: boolean
 }
 
-export type LegacyPriceQuoteParams = Omit<LegacyFeeQuoteParams, 'sellToken' | 'buyToken'> & PriceQuoteParams
+export type LegacyPriceQuoteParams = Omit<LegacyFeeQuoteParams, 'isEthFlow' | 'sellToken' | 'buyToken'> &
```
Comment (W3stside) [inline src/cow-react/api/gnosisProtocol/legacy/types.ts:38]: legacy doesn't accept eth anything, so we don't need it here

File: src/custom/hooks/useStablecoinPrice/index.ts [hunk 3]
```diff
@@ -105,6 +105,7 @@ export default function useCowUsdPrice(currency?: Currency) {
       toDecimals: stablecoin.decimals,
       userAddress: account,
       validTo: getUsdQuoteValidTo(),
+      isEthFlow: false,
```
Comment (W3stside) [inline src/custom/hooks/useStablecoinPrice/index.ts:108]: buy orders are never eth flow

File: src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts [hunk 4]
```diff
@@ -40,8 +40,8 @@ async function _getOrderPrice(chainId: ChainId, order: Order, strategy: GpPriceS
     quoteToken = order.sellToken
   }
 
-  const isNativeSellSwap = order.sellToken === NATIVE_CURRENCY_BUY_ADDRESS
-  if (isNativeSellSwap) {
+  const isEthFlow = order.sellToken === NATIVE_CURRENCY_BUY_ADDRESS
```
Comment (W3stside) [inline src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts:43]: we can't pass ethflow from state here as this checks orders in state that are not related to the current pair

File: src/cow-react/api/gnosisProtocol/api.ts (hunk 1 above)
Comment (W3stside) [reply src/cow-react/api/gnosisProtocol/api.ts]: 👍

**Comment by W3stside** [comment]
replacing with #1260
